dds/DCPS/XTypes/TypeAssignability.cpp

```diff
@@ -1067,6 +1067,15 @@ bool TypeAssignability::assignable_enum(const MinimalTypeObject& ta,
     return false;
   }
 
+  // T1.bit_bound and T2.bit_bound are in the same equivalence set
```

Comment:
```suggestion
  // T1.bit_bound and T2.bit_bound are in the same equivalence set (DDSXTY14-34)
```
provide tracking to this spec issue, since this is not yet actually part of the spec